trp_project_scrum/security/ir_rules_collaborators_followers.xml

```diff
@@ -0,0 +1,131 @@
+                ]</field>
+            <field name="groups" eval="[(4,ref('trp_project_scrum.collaborators_followers'))]"/>
+            <field name="global" eval="False" />
+            <field name="perm_unlink" eval="False" />
+            <field name="perm_write" eval="False" />
+            <field name="perm_read" eval="True" />
+            <field name="perm_create" eval="False" />
+        </record>
+
+        <record id="project_task_rule_foll" model="ir.rule">
+            <field name="name">Show Only collaborator Follower Task</field>
+            <field name="model_id" ref="project.model_project_task"/>
+            <field name="domain_force">[
+                    ('project_id.message_follower_ids', 'in', [user.partner_id.id]),
+                    ('use_scrum', '=', True)]</field>
+            <field name="groups" eval="[(4,ref('trp_project_scrum.collaborators_followers'))]"/>
+            <field name="global" eval="False" />
+            <field name="perm_unlink" eval="False" />
+            <field name="perm_write" eval="True" />
+            <field name="perm_read" eval="True" />
+            <field name="perm_create" eval="False" />
+        </record>
+
+        <record id="project_task_type_rule_foll" model="ir.rule">
+            <field name="name">Task Type collaborators followers</field>
+            <field name="model_id" ref="project.model_project_task_type"/>
+            <field name="groups" eval="[(4,ref('trp_project_scrum.collaborators_followers'))]"/>
+            <field name="global" eval="False" />
+            <field name="perm_unlink" eval="False" />
+            <field name="perm_write" eval="False" />
+            <field name="perm_read" eval="True" />
+            <field name="perm_create" eval="False" />
+        </record>
+
+        <record id="project_task_work_rule_foll" model="ir.rule">
+            <field name="name">All permissions on work Task for collaborators followers</field>
+            <field name="model_id" ref="project.model_project_task_work"/>
+            <field name="domain_force">[
+                    ('task_id.project_id.message_follower_ids', 'in', [user.partner_id.id]),
+                    ('task_id.project_id.use_scrum', '=', True)]</field>
+            <field name="groups" eval="[(4,ref('trp_project_scrum.collaborators_followers'))]"/>
+            <field name="global" eval="False" />
+            <field name="perm_unlink" eval="False" />
+            <field name="perm_write" eval="True" />
+            <field name="perm_read" eval="True" />
+            <field name="perm_create" eval="True" />
+        </record>
+
+        <record id="sprint_rule_foll" model="ir.rule">
+            <field name="name">Sprint for collaborators followers </field>
+            <field name="model_id" ref="project_scrum.model_project_scrum_sprint"/>
+            <field name="domain_force">
+                [('project_id.message_follower_ids', 'in', [user.partner_id.id])]</field>
```

Comment:
it's not coherent to assert `use_scrum` above and not here (for my taste, we shouldn't do it anywhere. why not allow to use this module to collaborate on non-scrum-projects?)